templates/appliance/hardware.html

```diff
@@ -0,0 +1,603 @@
+{% extends "appliance/_base-appliance.html" %}
+
+{% block title %}Recommended and Certified Hardware for Ubuntu Appliances{% endblock %}
+
+{% block meta_description %}We test every Ubuntu Appliance to work perfectly across all certified boards and boxes.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/179RBNgtGr5Y8CkkYaUa6sdFaaTJTTvs_VCTr6UzECFo/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <div class="col-10">
+      <h1>Recommended and Certified Hardware</h1>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block u-align--center">
+      <a href="#intel-nuc">
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/81dffc6e-Intel-header.jpg?h=150",
+            alt="",
+            height="150",
+            width="208",
+            hi_def=True,
+            loading="auto",
+            attrs={"style": "margin-bottom: .85rem;"},
```

Comment:
just asking, but should we u-hide--small here?

Reply:
Personally I liked the images here in the hero as its the main content leading down. The links look a little lost without an image.